schemas/proto/foxglove/CameraCalibration.proto

```diff
@@ -1,4 +1,4 @@
-// Generated from CameraCalibration by @foxglove/schemas
+// foxglove.CameraCalibration
```

Comment:
This comment is not useful. Its a proto file that has a `CameraCalibration` message in the `foxglove` namespace. Users of proto understand this equals `foxglove.CameraCalibration`.

The comment should mention what repo generated this and could also link to our MessageSchema docs which help you discover how to then use them in studio.

Reply:
removed for proto files